vortex-datafusion/src/persistent/format.rs

```diff
@@ -136,6 +136,7 @@ impl FileFormat for VortexFormat {
             stats.column_statistics = column_statistics;
         }
 
+        log::info!("Inferred stats: {:?}", stats);
```

Comment:
do we actually need this?

Reply:
main issue is that the benchmark bin shouldn't write anything to stdout, so this might break it, honestly not sure how `log` is configured.